esm-samples/jsapi-node/rollup.config.js

```diff
@@ -1,14 +1,25 @@
 import commonjs from "@rollup/plugin-commonjs";
+import copy from "rollup-plugin-copy";
+import del from "rollup-plugin-delete";
 import resolve from "@rollup/plugin-node-resolve";
 
 export default {
   input: ["src/request.js", "src/projection.js", "src/webmap.js"],
   output: {
     chunkFileNames: "chunks/[name].js",
-    dir: "dist",
+    dir: "public",
     format: "cjs"
   },
   external: ["whatwg-fetch"],
-  plugins: [resolve(), commonjs()],
+  plugins: [
+    del({ targets: ["public/chunks", "public/assets"], runOnce: true, verbose: true }),
```

Comment:
Can it delete the whole `public` dir?

Reply:
Yes, I believe it can.

Reply:
Done via `del({ targets: ["public/*"], runOnce: true, verbose: true }),`